distribution/resources/cli-conf/toolkit-config.toml

```diff
@@ -3,8 +3,8 @@ httpRequestTimeout = 1000000
 
 [token]
 baseURL = ""
-restVersion = "v1.1"
-dcrVersion = "v0.16"
+restVersion = "v1.2"
```

Comment:
Shall we updated the default rest and dcr version in here as well. https://github.com/wso2/product-microgateway/blob/9603fbd554d200cca33c10114f70232dbe9b7b6c/components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/constants/RESTServiceConstants.java#L47

Reply:
fixed with : ba4c1481c18b7e6da52a09bc16141f69f674c3cc